pyaedt/modeler/cad/Primitives.py

```diff
@@ -1372,12 +1370,25 @@ def refresh(self):
         self._all_object_names = []
         self.objects = {}
         self.user_defined_components = {}
-        self.object_id_dict = {}
+        self._object_id_to_names = {}
         self._currentId = 0
         self._refresh_object_types()
         self._refresh_all_ids_from_aedt_file()
         self.refresh_all_ids()
 
+    @property
+    def objects_by_name(self):
+        """Object dictionary organized by name.
+
+        Returns
+        -------
+        dict
+        """
+        obj_dict = {}
+        for _, v in self.objects.items():
```

Comment:
why not like this?

```suggestion
        for v in self.objects.values():
```

Reply:
because for Ironpython I should use list(self.objects.values()). I prefer this approach